contracts/DexClassic.sol

```diff
@@ -12,8 +12,9 @@ contract DexClassic is Ownable {
     AggregatorV3Interface public priceFeed; 
 
     constructor(ERC20 usdToken) Ownable() {
-        usd = usdToken;        
-        priceFeed = AggregatorV3Interface("YOUR_DEPLOYED_PRICE_FEED_CONTRACT_ADDRESS");        
+        // You should replace address(0) with your deployed PriceFeed address
```

Comment:
I changed this so that contracts can be compiled without any changes in the repo

Reply:
Ok, I will revert